cpp/src/distance/distance/specializations/detail/canberra_double_double_double_int.cu

```diff
@@ -19,20 +19,8 @@
 namespace raft {
 namespace distance {
 namespace detail {
-template void distance<raft::distance::DistanceType::Canberra, float, float, float, int>(
-  const float* x,
```

Comment:
Oh wow, so we were building the same canberra specialization multiple times? It makes sense now...